php/commands/src/Alias_Command.php

```diff
@@ -0,0 +1,268 @@
+<?php
+
+use Mustangostang\Spyc;
+
+/**
+ * Retrieves and sets alias for WP Installs.
+ *
+ * Aliases are shorthand references to WordPress installs. For instance,
+ * `@dev` could refer to a development install and `@prod` could refer to
+ * a production install. This command gives you visibility in what
+ * registered aliases you have available.
+ *
+ * ## EXAMPLES
+ *
+ *     # Get alias information.
+ *     $ wp alias get @dev
```

Comment:
These examples need to be `wp cli alias` now I guess 🙂

Reply:
Ah 😄